### PR TITLE
chore: Cleanup various files and refactor 

### PR DESCRIPTION
**Pull request by samuveth**
### Summary

See comments below

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -1,62 +1,13 @@
...
-import AutoImport from 'unplugin-auto-import/vite';
+import { defineConfig } from 'vitest/config';
+import { getPlugins } from './src/helpers/vitePlugins';
 
 export default defineConfig({
   define: {
     'process.env': process.env
   },
-  plugins: [
-    vue({ reactivityTransform: true }),
```
Comment (samuveth) [inline vite.config.ts:18]: Moved plugins into `vitePlugins.ts` helper to keep this file more readable

File: src/main.ts [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
-import router from '@/router';
-import i18n from '@/helpers/i18n';
-import '@/helpers/auth';
-import '@/assets/css/main.scss';
-import { apolloClient } from '@/helpers/apollo';
 import { DefaultApolloClient } from '@vue/apollo-composable';
-import VueTippy from 'vue-tippy';
 import { createHead } from '@vueuse/head';
-import 'viewerjs/dist/viewer.css';
+import options from '@/helpers/auth';
```
Comment (samuveth) [inline src/main.ts:8]: Organized by internal import at the top and external imports below

File: src/main.ts [hunk 3]
```diff
@@ -1,25 +1,22 @@
 import { Buffer } from 'buffer';
 (window as any).global = window;
 (window as any).Buffer = Buffer;
+
 import { LockPlugin } from '@snapshot-labs/lock/plugins/vue3';
-import options from '@/helpers/auth';
-import '../snapshot-spaces/skins';
```
Comment (samuveth) [inline src/main.ts:6]: Moved into `main.scss`

File: vite.config.ts [hunk 4]
```diff
@@ -1,62 +1,13 @@
...
-import { defineConfig } from 'vite';
-import vue from '@vitejs/plugin-vue';
-import ViteComponents from 'unplugin-vue-components/vite';
-import visualizer from 'rollup-plugin-visualizer';
-import Icons from 'unplugin-icons/vite';
-import IconsResolver from 'unplugin-icons/resolver';
-import { FileSystemIconLoader } from 'unplugin-icons/loaders';
-import { sentryVitePlugin } from '@sentry/vite-plugin';
-import AutoImport from 'unplugin-auto-import/vite';
+import { defineConfig } from 'vitest/config';
```
Comment (samuveth) [inline vite.config.ts:3]: Import from `vitest/config` instead of `vite` to support test typing

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**